app/_locales/en/messages.json

```diff
@@ -626,6 +626,9 @@
   "buy": {
     "message": "Buy"
   },
+  "buyAndSell": {
```

Comment:
You might also want to take a look at `ui/components/multichain/select-action-modal/select-action-modal.js` and replace the text there :)

Reply:
Great catch! I will need to also request copywriting for the description in that component. Will update soon. Thank you Danica

Reply:
Added in [16f95de](https://github.com/MetaMask/metamask-extension/pull/20891/commits/16f95de8894c345b2862d7c8f0ca66ee593fc371)